src/System.EventLog/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netfx;
+      netcoreapp;
```

Comment:
I think I mentioned in the other fork, but this shoudl be `netcoreapp-Windows_NT` so the tests don't try to run on Unix.

Reply:
done